lib/CollectionsTable/styles.scss

```diff
@@ -133,6 +133,7 @@ $collections-table-cell-min-height: 40px !default;
 .collections-table__row-title {
   margin: 0;
   font-size: $typo-size-base;
+  font-weight: 400;
```

Comment:
`$typo-weight-roman` is the var we use to specify a weight of 400. You can see these settings in `styles/settings/_typography.scss`.